sims/verilator/Makefile

```diff
@@ -56,20 +56,29 @@ model_mk_debug = $(model_dir_debug)/V$(VLOG_MODEL).mk
 # build makefile fragment that builds the verilator sim rules
 #########################################################################################
 LDFLAGS := $(LDFLAGS) -L$(RISCV)/lib -Wl,-rpath,$(RISCV)/lib -L$(sim_dir) -lfesvr -lpthread
+SHARED_FLAGS := \
```

Comment:
How did you decide what to include in this variable? It appears there are several other "shared" flags between the two invocations. Why not go all in?

Reply:
Oh. So I could put in the `VERILATOR_FLAGS` into the `SHARED_FLAGS`. However, the other flags are different (different directories, one has `--trace`, different `-o` files, different `-include`s). I can make this cleaner if you want.

Reply:
Yeah I'd also consider having two `-CFLAGS` arguments so that the shared flags can also absorb the first two flags there.